### ISSUE TITLE
Add more flight recorder events

### ISSUE THREAD

**Issue by patriknw** [akka/akka#21516]
e.g. stream materialization/restart
look for places where we log and see if that should also be an AFR event

**Comment by patriknw** [comment]
I tried to use AFR when debugging the stuck heartbeating and one thing I'm missing is the ability to see what channel/stream events from AeronSource/AeronSink belong to. The low freq events contains the channel, (`eron:udp?endpoint=ip-172-31-8-204:25520`) but it should also include the  stream. The value of the high freq is close to zero without this info and if that is too much overhead to add, then we should replace those with counters.

**Comment by drewhk** [comment]
Yep, we can add more fields there. It is not about overhead, I told before about adding one more distinguisher field, it just never got added due to no particular reason.
